### ISSUE TITLE
Error when sharing dataset on testhub

### ISSUE THREAD

**Issue by HeDo88TH** [DroneDB/Registry#147]
Probable regression of #142 

`ddb share -t hedo88/mont *`

```
2021-04-28T11:55:33.0943412+02:00 0HM89PLJPV213:00000004 [INF] Opened ddb in './ddbstore/hedo88/8bfefc87-95fb-41b1-8715-acf62bc4d691' (9008bfba)
2021-04-28T11:55:33.0956267+02:00 0HM89PLJPV213:00000004 [ERR] Exception in Share controller Upload('kbDAESaewChPDKS8UVUa8yb29n4owJYg', 'IMG_20160904_130800.jpg', 'IMG_20160904_130800.jpg') (59aebb46)
System.InvalidOperationException: The inner stream position has changed unexpectedly.
   at Microsoft.AspNetCore.Http.ReferenceReadStream.VerifyPosition()
   at Microsoft.AspNetCore.Http.ReferenceReadStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.Stream.CopyTo(Stream destination, Int32 bufferSize)
   at Registry.Adapters.DroneDB.Ddb.Add(String path, Stream stream) in /Registry/Registry.Adapters.DroneDB/Ddb.cs:line 200
   at Registry.Web.Services.Managers.ObjectsManager.AddNew(String orgSlug, String dsSlug, String path, Stream stream) in /Registry/Registry.Web/Services/Managers/ObjectsManager.cs:line 212
   at Registry.Web.Services.Managers.ShareManager.Upload(String token, String path, Stream stream) in /Registry/Registry.Web/Services/Managers/ShareManager.cs:line 473
   at Registry.Web.Controllers.ShareController.Upload(String token, String path, IFormFile file) in /Registry/Registry.Web/Controllers/ShareController.cs:line 73
   at Registry.Web.Controllers.ShareController.Upload(String token, String path, IFormFile file) in /Registry/Registry.Web/Controllers/ShareController.cs:line 74
```

**Comment by pierotofy** [comment]
Can confirm:

```
[piero:/data … righton2/images] $ ddb share -t pierotofy/test DJI_0018.JPG 
Username: pierotofy
Password: 
Error response from registry: The inner stream position has changed unexpectedly.
```

**Comment by pierotofy** [comment]
Maybe due to https://github.com/DroneDB/Registry/commit/83851b7d3e7b21dd97725c0a4164e8ec0c5282c0 ?

It works locally though.

**Comment by HeDo88TH** [comment]
> Maybe due to [83851b7](https://github.com/DroneDB/Registry/commit/83851b7d3e7b21dd97725c0a4164e8ec0c5282c0) ?
> 
> It works locally though.
> 
> ```
> ddb share -t http://192.168.2.251:5000/admin/test DJI_0018.JPG 
> DJI_0018.JPG [###########################] 100.00% 3.14s
> http://192.168.2.251:5000/r/admin/test
> ```

I'm trying to reproduce the issue locally too but without success. It works on both windows and linux

**Comment by HeDo88TH** [comment]
I succedeed in reproducing the issue. It shows up when using inside docker, I think it could be something related to it.
I keep digging.

**Comment by HeDo88TH** [comment]
Fixed in #148
